test/test.js

```diff
@@ -21289,9 +21357,13 @@
       }, 192);
 
       setTimeout(function() {
+        assert.strictEqual(callCount, 1);
```

Comment:
I changed the timings of the tests.  The commit message for this change has a bit more details: 43d71efd6e56d9f4041434737fe8b3286b780296